AirCasting/AuthViews/UserSettings.swift

```diff
@@ -6,15 +6,18 @@ import Foundation
 class UserSettings: ObservableObject {
     private let userDefaults: UserDefaults
     
-    @Published var contributingToCrowdMap: Bool {
-        didSet {
-            Log.info("Changed crowdmap contribution setting to \(contributingToCrowdMap ? "ON" : "OFF")")
-            userDefaults.setValue(contributingToCrowdMap, forKey: "crowdMap")
+    var contributingToCrowdMap: Bool {
+        get {
+            userDefaults.bool(forKey: "crowdMap")
```

Comment:
Why not use @AppStorage here? :)

Reply:
We have (I think) about that with Paweł that then it is somehow directly access to UserDefeults and now not exactly but I am not sure and I am learning so don't listen to me. We need to ask @pawelgil